modules/graphql_views/src/Plugin/Deriver/ViewFilterInputDeriver.php

```diff
@@ -58,4 +62,56 @@ public function getDerivativeDefinitions($basePluginDefinition) {
     return parent::getDerivativeDefinitions($basePluginDefinition);
   }
 
+
+  public function isGenericInputFilter($filter) {
+    if (!is_array($filter['value'])) {
+      return false;
+    }
+
+    if (count($filter['value']) == 0) {
+      return false;
+    }
+
+    return is_string( array_keys($filter['value'] )[0] );
```

Comment:
So basically combine the if() from line 71 and the return on line 75 in one conditional.